[Automation] javac compile fails for org.seleniumhq.selenium:selenium-manager:4.11.0

## javac compile on ubuntu-latest / JDK 25 / mode current-defaults

Failure kind: javac compile
OS: ubuntu-latest
JDK: 25
Native-image mode: current-defaults
Reproducer: `GVM_TCK_NATIVE_IMAGE_MODE="current-defaults" GVM_TCK_LV="4.11.0" ./gradlew clean compileTestJava -Pcoordinates="org.seleniumhq.selenium:selenium-manager:4.9.0"`
Runner log: https://github.com/oracle/graalvm-reachability-metadata/actions/runs/25541257498/job/74967784701
Last 300 lines of the log:
```console
[         3 containers found      ]
[         0 containers skipped    ]
[         3 containers started    ]
[         0 containers aborted    ]
[         3 containers successful ]
[         0 containers failed     ]
[         1 tests found           ]
[         0 tests skipped         ]
[         1 tests started         ]
[         0 tests aborted         ]
[         1 tests successful      ]
[         0 tests failed          ]


[Incubating] Problems report is available at: file:///home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/tests/src/org.seleniumhq.selenium/selenium-manager/4.9.0/build/reports/problems/problems-report.html

Deprecated Gradle features were used in this build, making it incompatible with Gradle 10.

You can use '--warning-mode all' to show the individual deprecation warnings and determine if they come from your own scripts or plugins.

For more on this, please refer to https://docs.gradle.org/9.1.0/userguide/command_line_interface.html#sec:command_line_warnings in the Gradle documentation.

BUILD SUCCESSFUL in 1m 23s
17 actionable tasks: 7 executed, 10 up-to-date
Consider enabling configuration cache to speed up this build: https://docs.gradle.org/9.1.0/userguide/configuration_cache_enabling.html
-------
Test for org.seleniumhq.selenium:selenium-manager:4.9.0 passed.
====================

> Task :test

[Incubating] Problems report is available at: file:///home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/build/reports/problems/problems-report.html

Deprecated Gradle features were used in this build, making it incompatible with Gradle 10.

You can use '--warning-mode all' to show the individual deprecation warnings and determine if they come from your own scripts or plugins.

For more on this, please refer to https://docs.gradle.org/9.1.0/userguide/command_line_interface.html#sec:command_line_warnings in the Gradle documentation.

BUILD SUCCESSFUL in 1m 24s
11 actionable tasks: 3 executed, 8 up-to-date
Finished stage 'native-image run' for 4.10.0 at 2026-05-08T06:49:36Z with exit code 0 after 85s
PASSED:4.10.0
========================================================================================
 Testing org.seleniumhq.selenium:selenium-manager:4.9.0:4.11.0
========================================================================================
========================================================================================
 org.seleniumhq.selenium:selenium-manager:4.9.0:4.11.0 stage native-image run
========================================================================================
Starting stage 'native-image run' for 4.11.0 at 2026-05-08T06:49:36Z
Command: GVM_TCK_NATIVE_IMAGE_MODE="current-defaults" GVM_TCK_LV="4.11.0" ./gradlew clean test -Pcoordinates="org.seleniumhq.selenium:selenium-manager:4.9.0"
> Task :tck-build-logic:extractPluginRequests UP-TO-DATE
> Task :tck-build-logic:generatePluginAdapters UP-TO-DATE
> Task :tck-build-logic:compileJava UP-TO-DATE
> Task :tck-build-logic:compileGroovy UP-TO-DATE
> Task :tck-build-logic:compileGroovyPlugins UP-TO-DATE
> Task :tck-build-logic:pluginDescriptors UP-TO-DATE
> Task :tck-build-logic:processResources UP-TO-DATE
> Task :tck-build-logic:classes UP-TO-DATE
> Task :tck-build-logic:jar UP-TO-DATE

> Configure project :
GraalVM Reachability Metadata TCK
---------------------------------

> Task :tckClean
> Task :tck-build-logic:extractPluginRequests UP-TO-DATE
> Task :tck-build-logic:generatePluginAdapters UP-TO-DATE
> Task :tck-build-logic:compileJava UP-TO-DATE
> Task :tck-build-logic:compileGroovy UP-TO-DATE
> Task :tck-build-logic:compileGroovyPlugins UP-TO-DATE
> Task :tck-build-logic:pluginDescriptors UP-TO-DATE
> Task :tck-build-logic:processResources UP-TO-DATE
> Task :tck-build-logic:classes UP-TO-DATE
> Task :tck-build-logic:jar UP-TO-DATE
> Task :clean

[Incubating] Problems report is available at: file:///home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/tests/src/org.seleniumhq.selenium/selenium-manager/4.9.0/build/reports/problems/problems-report.html

Deprecated Gradle features were used in this build, making it incompatible with Gradle 10.

You can use '--warning-mode all' to show the individual deprecation warnings and determine if they come from your own scripts or plugins.

For more on this, please refer to https://docs.gradle.org/9.1.0/userguide/command_line_interface.html#sec:command_line_warnings in the Gradle documentation.

BUILD SUCCESSFUL in 817ms
9 actionable tasks: 1 executed, 8 up-to-date
Consider enabling configuration cache to speed up this build: https://docs.gradle.org/9.1.0/userguide/configuration_cache_enabling.html

> Task :clean

> Task :tckTest FAILED
====================
Testing library: org.seleniumhq.selenium:selenium-manager:4.9.0
Command: `/home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/gradlew nativeTest`
Executing test...
-------
> Task :tck-build-logic:extractPluginRequests UP-TO-DATE
> Task :tck-build-logic:generatePluginAdapters UP-TO-DATE
> Task :tck-build-logic:compileJava UP-TO-DATE
> Task :tck-build-logic:compileGroovy UP-TO-DATE
> Task :tck-build-logic:compileGroovyPlugins UP-TO-DATE
> Task :tck-build-logic:pluginDescriptors UP-TO-DATE
> Task :tck-build-logic:processResources UP-TO-DATE
> Task :tck-build-logic:classes UP-TO-DATE
> Task :tck-build-logic:jar UP-TO-DATE
> Task :compileJava NO-SOURCE
> Task :processResources NO-SOURCE
> Task :classes UP-TO-DATE
> Task :tck-test-support:compileJava UP-TO-DATE
> Task :tck-test-support:processResources NO-SOURCE
> Task :tck-test-support:classes UP-TO-DATE
> Task :tck-test-support:jar UP-TO-DATE
> Task :jar

> Task :compileTestJava FAILED
/home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/tests/src/org.seleniumhq.selenium/selenium-manager/4.9.0/src/test/java/org_seleniumhq_selenium/selenium_manager/SeleniumManagerTest.java:23: error: method getDriverPath in class SeleniumManager cannot be applied to given types;
        assertThatThrownBy(() -> SeleniumManager.getInstance().getDriverPath(capabilities))
                                                              ^
  required: Capabilities,boolean
[Incubating] Problems report is available at: file:///home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/tests/src/org.seleniumhq.selenium/selenium-manager/4.9.0/build/reports/problems/problems-report.html

  found:    ImmutableCapabilities
  reason: actual and formal argument lists differ in length
1 error

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':compileTestJava'.
> Compilation failed; see the compiler output below.
  /home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/tests/src/org.seleniumhq.selenium/selenium-manager/4.9.0/src/test/java/org_seleniumhq_selenium/selenium_manager/SeleniumManagerTest.java:23: error: method getDriverPath in class SeleniumManager cannot be applied to given types;
          assertThatThrownBy(() -> SeleniumManager.getInstance().getDriverPath(capabilities))
                                                                ^
    required: Capabilities,boolean
    found:    ImmutableCapabilities
    reason: actual and formal argument lists differ in length
  1 error

* Try:
> 
Deprecated Gradle features were used in this build, making it incompatible with Gradle 10.

You can use '--warning-mode all' to show the individual deprecation warnings and determine if they come from your own scripts or plugins.

For more on this, please refer to https://docs.gradle.org/9.1.0/userguide/command_line_interface.html#sec:command_line_warnings in the Gradle documentation.
12 actionable tasks: 2 executed, 10 up-to-date
Check your code and dependencies to fix the compilation error(s)
> Run with --scan to generate a Build Scan (Powered by Develocity).

BUILD FAILED in 1s

[Incubating] Problems report is available at: file:///home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/build/reports/problems/problems-report.html

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':tckTest'.
> Test for org.seleniumhq.selenium:selenium-manager:4.9.0 failed with exit code 1.

* Try:
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to generate a Build Scan (Powered by Develocity).
> Get more help at https://help.gradle.org.

Deprecated Gradle features were used in this build, making it incompatible with Gradle 10.

You can use '--warning-mode all' to show the individual deprecation warnings and determine if they come from your own scripts or plugins.

For more on this, please refer to https://docs.gradle.org/9.1.0/userguide/command_line_interface.html#sec:command_line_warnings in the Gradle documentation.

BUILD FAILED in 3s
11 actionable tasks: 3 executed, 8 up-to-date
Finished stage 'native-image run' for 4.11.0 at 2026-05-08T06:49:40Z with exit code 1 after 4s
❌ ERROR: 'native-image run' for 4.11.0 failed with exit code 1.
SOFT-FAIL[native-image run][4.11.0][GVM_TCK_NATIVE_IMAGE_MODE="current-defaults" GVM_TCK_LV="4.11.0" ./gradlew clean test -Pcoordinates="org.seleniumhq.selenium:selenium-manager:4.9.0"]
========================================================================================
 org.seleniumhq.selenium:selenium-manager:4.9.0:4.11.0 stage javac compile
========================================================================================
Starting stage 'javac compile' for 4.11.0 at 2026-05-08T06:49:40Z
Command: GVM_TCK_NATIVE_IMAGE_MODE="current-defaults" GVM_TCK_LV="4.11.0" ./gradlew clean compileTestJava -Pcoordinates="org.seleniumhq.selenium:selenium-manager:4.9.0"
> Task :tck-build-logic:extractPluginRequests UP-TO-DATE
> Task :tck-build-logic:generatePluginAdapters UP-TO-DATE
> Task :tck-build-logic:compileJava UP-TO-DATE
> Task :tck-build-logic:compileGroovy UP-TO-DATE
> Task :tck-build-logic:compileGroovyPlugins UP-TO-DATE
> Task :tck-build-logic:pluginDescriptors UP-TO-DATE
> Task :tck-build-logic:processResources UP-TO-DATE
> Task :tck-build-logic:classes UP-TO-DATE
> Task :tck-build-logic:jar UP-TO-DATE

> Configure project :
GraalVM Reachability Metadata TCK
---------------------------------

> Task :tckClean
> Task :tck-build-logic:extractPluginRequests UP-TO-DATE
> Task :tck-build-logic:generatePluginAdapters UP-TO-DATE
> Task :tck-build-logic:compileJava UP-TO-DATE
> Task :tck-build-logic:compileGroovy UP-TO-DATE
> Task :tck-build-logic:compileGroovyPlugins UP-TO-DATE
> Task :tck-build-logic:pluginDescriptors UP-TO-DATE
> Task :tck-build-logic:processResources UP-TO-DATE
> Task :tck-build-logic:classes UP-TO-DATE
> Task :tck-build-logic:jar UP-TO-DATE
> Task :clean

[Incubating] Problems report is available at: file:///home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/tests/src/org.seleniumhq.selenium/selenium-manager/4.9.0/build/reports/problems/problems-report.html

Deprecated Gradle features were used in this build, making it incompatible with Gradle 10.

You can use '--warning-mode all' to show the individual deprecation warnings and determine if they come from your own scripts or plugins.

For more on this, please refer to https://docs.gradle.org/9.1.0/userguide/command_line_interface.html#sec:command_line_warnings in the Gradle documentation.

BUILD SUCCESSFUL in 769ms
9 actionable tasks: 1 executed, 8 up-to-date
Consider enabling configuration cache to speed up this build: https://docs.gradle.org/9.1.0/userguide/configuration_cache_enabling.html

> Task :clean

> Task :compileTestJava FAILED
> Task :tck-build-logic:extractPluginRequests UP-TO-DATE
> Task :tck-build-logic:generatePluginAdapters UP-TO-DATE
> Task :tck-build-logic:compileJava UP-TO-DATE
> Task :tck-build-logic:compileGroovy UP-TO-DATE
> Task :tck-build-logic:compileGroovyPlugins UP-TO-DATE
> Task :tck-build-logic:pluginDescriptors UP-TO-DATE
> Task :tck-build-logic:processResources UP-TO-DATE
> Task :tck-build-logic:classes UP-TO-DATE
> Task :tck-build-logic:jar UP-TO-DATE
> Task :compileJava NO-SOURCE
> Task :tck-test-support:compileJava UP-TO-DATE
> Task :processResources NO-SOURCE
> Task :classes UP-TO-DATE

> Task :compileTestJava FAILED
/home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/tests/src/org.seleniumhq.selenium/selenium-manager/4.9.0/src/test/java/org_seleniumhq_selenium/selenium_manager/SeleniumManagerTest.java:23: error: method getDriverPath in class SeleniumManager cannot be applied to given types;
        assertThatThrownBy(() -> SeleniumManager.getInstance().getDriverPath(capabilities))
                                                              ^
  required: Capabilities,boolean
  found:    ImmutableCapabilities
  reason: actual and formal argument lists differ in length

[Incubating] Problems report is available at: file:///home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/tests/src/org.seleniumhq.selenium/selenium-manager/4.9.0/build/reports/problems/problems-report.html
1 error

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':compileTestJava'.
> Compilation failed; see the compiler output below.
  /home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/tests/src/org.seleniumhq.selenium/selenium-manager/4.9.0/src/test/java/org_seleniumhq_selenium/selenium_manager/SeleniumManagerTest.java:23: error: method getDriverPath in class SeleniumManager cannot be applied to given types;
          assertThatThrownBy(() -> SeleniumManager.getInstance().getDriverPath(capabilities))
                                                                ^
    required: Capabilities,boolean
    found:    ImmutableCapabilities

Deprecated Gradle features were used in this build, making it incompatible with Gradle 10.

You can use '--warning-mode all' to show the individual deprecation warnings and determine if they come from your own scripts or plugins.

    reason: actual and formal argument lists differ in length
  1 error

* Try:
> Check your code and dependencies to fix the compilation error(s)
> Run with --scan to generate a Build Scan (Powered by Develocity).

BUILD FAILED in 881ms
For more on this, please refer to https://docs.gradle.org/9.1.0/userguide/command_line_interface.html#sec:command_line_warnings in the Gradle documentation.
10 actionable tasks: 1 executed, 9 up-to-date

[Incubating] Problems report is available at: file:///home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/build/reports/problems/problems-report.html

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':compileTestJava'.
> Compilation failed

* Try:
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to generate a Build Scan (Powered by Develocity).
> Get more help at https://help.gradle.org.

Deprecated Gradle features were used in this build, making it incompatible with Gradle 10.

You can use '--warning-mode all' to show the individual deprecation warnings and determine if they come from your own scripts or plugins.

For more on this, please refer to https://docs.gradle.org/9.1.0/userguide/command_line_interface.html#sec:command_line_warnings in the Gradle documentation.

BUILD FAILED in 2s
11 actionable tasks: 3 executed, 8 up-to-date
Finished stage 'javac compile' for 4.11.0 at 2026-05-08T06:49:43Z with exit code 1 after 3s
❌ ERROR: 'javac compile' for 4.11.0 failed with exit code 1.
FAILED[javac compile][4.11.0][GVM_TCK_NATIVE_IMAGE_MODE="current-defaults" GVM_TCK_LV="4.11.0" ./gradlew clean compileTestJava -Pcoordinates="org.seleniumhq.selenium:selenium-manager:4.9.0"]
```

## javac compile on ubuntu-latest / JDK latest-ea / mode current-defaults

Failure kind: javac compile
OS: ubuntu-latest
JDK: latest-ea
Native-image mode: current-defaults
Reproducer: `GVM_TCK_NATIVE_IMAGE_MODE="current-defaults" GVM_TCK_LV="4.11.0" ./gradlew clean compileTestJava -Pcoordinates="org.seleniumhq.selenium:selenium-manager:4.9.0"`
Runner log: https://github.com/oracle/graalvm-reachability-metadata/actions/runs/25541257498/job/74967784596
Last 300 lines of the log:
```console
[         3 containers found      ]
[         0 containers skipped    ]
[         3 containers started    ]
[         0 containers aborted    ]
[         3 containers successful ]
[         0 containers failed     ]
[         1 tests found           ]
[         0 tests skipped         ]
[         1 tests started         ]
[         0 tests aborted         ]
[         1 tests successful      ]
[         0 tests failed          ]


[Incubating] Problems report is available at: file:///home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/tests/src/org.seleniumhq.selenium/selenium-manager/4.9.0/build/reports/problems/problems-report.html

Deprecated Gradle features were used in this build, making it incompatible with Gradle 10.

You can use '--warning-mode all' to show the individual deprecation warnings and determine if they come from your own scripts or plugins.

For more on this, please refer to https://docs.gradle.org/9.1.0/userguide/command_line_interface.html#sec:command_line_warnings in the Gradle documentation.

BUILD SUCCESSFUL in 1m 26s
17 actionable tasks: 7 executed, 10 up-to-date
Consider enabling configuration cache to speed up this build: https://docs.gradle.org/9.1.0/userguide/configuration_cache_enabling.html
-------
Test for org.seleniumhq.selenium:selenium-manager:4.9.0 passed.
====================

> Task :test

[Incubating] Problems report is available at: file:///home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/build/reports/problems/problems-report.html

Deprecated Gradle features were used in this build, making it incompatible with Gradle 10.

You can use '--warning-mode all' to show the individual deprecation warnings and determine if they come from your own scripts or plugins.

For more on this, please refer to https://docs.gradle.org/9.1.0/userguide/command_line_interface.html#sec:command_line_warnings in the Gradle documentation.

BUILD SUCCESSFUL in 1m 28s
11 actionable tasks: 3 executed, 8 up-to-date
Finished stage 'native-image run' for 4.10.0 at 2026-05-08T06:49:53Z with exit code 0 after 88s
PASSED:4.10.0
========================================================================================
 Testing org.seleniumhq.selenium:selenium-manager:4.9.0:4.11.0
========================================================================================
========================================================================================
 org.seleniumhq.selenium:selenium-manager:4.9.0:4.11.0 stage native-image run
========================================================================================
Starting stage 'native-image run' for 4.11.0 at 2026-05-08T06:49:53Z
Command: GVM_TCK_NATIVE_IMAGE_MODE="current-defaults" GVM_TCK_LV="4.11.0" ./gradlew clean test -Pcoordinates="org.seleniumhq.selenium:selenium-manager:4.9.0"
> Task :tck-build-logic:extractPluginRequests UP-TO-DATE
> Task :tck-build-logic:generatePluginAdapters UP-TO-DATE
> Task :tck-build-logic:compileJava UP-TO-DATE
> Task :tck-build-logic:compileGroovy UP-TO-DATE
> Task :tck-build-logic:compileGroovyPlugins UP-TO-DATE
> Task :tck-build-logic:pluginDescriptors UP-TO-DATE
> Task :tck-build-logic:processResources UP-TO-DATE
> Task :tck-build-logic:classes UP-TO-DATE
> Task :tck-build-logic:jar UP-TO-DATE

> Configure project :
GraalVM Reachability Metadata TCK
---------------------------------

> Task :tckClean
> Task :tck-build-logic:extractPluginRequests UP-TO-DATE
> Task :tck-build-logic:generatePluginAdapters UP-TO-DATE
> Task :tck-build-logic:compileJava UP-TO-DATE
> Task :tck-build-logic:compileGroovy UP-TO-DATE
> Task :tck-build-logic:compileGroovyPlugins UP-TO-DATE
> Task :tck-build-logic:pluginDescriptors UP-TO-DATE
> Task :tck-build-logic:processResources UP-TO-DATE
> Task :tck-build-logic:classes UP-TO-DATE
> Task :tck-build-logic:jar UP-TO-DATE
> Task :clean

[Incubating] Problems report is available at: file:///home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/tests/src/org.seleniumhq.selenium/selenium-manager/4.9.0/build/reports/problems/problems-report.html

Deprecated Gradle features were used in this build, making it incompatible with Gradle 10.

You can use '--warning-mode all' to show the individual deprecation warnings and determine if they come from your own scripts or plugins.

For more on this, please refer to https://docs.gradle.org/9.1.0/userguide/command_line_interface.html#sec:command_line_warnings in the Gradle documentation.

BUILD SUCCESSFUL in 873ms
9 actionable tasks: 1 executed, 8 up-to-date
Consider enabling configuration cache to speed up this build: https://docs.gradle.org/9.1.0/userguide/configuration_cache_enabling.html

> Task :clean

> Task :tckTest FAILED
====================
Testing library: org.seleniumhq.selenium:selenium-manager:4.9.0
Command: `/home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/gradlew nativeTest`
Executing test...
-------
> Task :tck-build-logic:extractPluginRequests UP-TO-DATE
> Task :tck-build-logic:generatePluginAdapters UP-TO-DATE
> Task :tck-build-logic:compileJava UP-TO-DATE
> Task :tck-build-logic:compileGroovy UP-TO-DATE
> Task :tck-build-logic:compileGroovyPlugins UP-TO-DATE
> Task :tck-build-logic:pluginDescriptors UP-TO-DATE
> Task :tck-build-logic:processResources UP-TO-DATE
> Task :tck-build-logic:classes UP-TO-DATE
> Task :tck-build-logic:jar UP-TO-DATE
> Task :compileJava NO-SOURCE
> Task :tck-test-support:compileJava UP-TO-DATE
> Task :processResources NO-SOURCE
> Task :classes UP-TO-DATE
> Task :tck-test-support:processResources NO-SOURCE
> Task :tck-test-support:classes UP-TO-DATE
> Task :tck-test-support:jar UP-TO-DATE
> Task :jar

> Task :compileTestJava FAILED
/home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/tests/src/org.seleniumhq.selenium/selenium-manager/4.9.0/src/test/java/org_seleniumhq_selenium/selenium_manager/SeleniumManagerTest.java:23: error: method getDriverPath in class SeleniumManager cannot be applied to given types;
        assertThatThrownBy(() -> SeleniumManager.getInstance().getDriverPath(capabilities))
                                                              ^
  required: Capabilities,boolean
  found:    ImmutableCapabilities
  reason: actual and formal argument lists differ in length
1 error

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':compileTestJava'.
> Compilation failed; see the compiler output below.
  /home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/tests/src/org.seleniumhq.selenium/selenium-manager/4.9.0/src/test/java/org_seleniumhq_selenium/selenium_manager/SeleniumManagerTest.java:23: error: method getDriverPath in class SeleniumManager cannot be applied to given types;
          assertThatThrownBy(() -> SeleniumManager.getInstance().getDriverPath(capabilities))
                                                                ^
    required: Capabilities,boolean
    found:    ImmutableCapabilities
    reason: actual and formal argument lists differ in length
  1 error


[Incubating] Problems report is available at: file:///home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/tests/src/org.seleniumhq.selenium/selenium-manager/4.9.0/build/reports/problems/problems-report.html

Deprecated Gradle features were used in this build, making it incompatible with Gradle 10.

You can use '--warning-mode all' to show the individual deprecation warnings and determine if they come from your own scripts or plugins.

For more on this, please refer to https://docs.gradle.org/9.1.0/userguide/command_line_interface.html#sec:command_line_warnings in the Gradle documentation.* Try:
> Check your code and dependencies to fix the compilation error(s)
> Run with --scan to generate a Build Scan (Powered by Develocity).

BUILD FAILED in 1s

12 actionable tasks: 2 executed, 10 up-to-date

[Incubating] Problems report is available at: file:///home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/build/reports/problems/problems-report.html

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':tckTest'.
> Test for org.seleniumhq.selenium:selenium-manager:4.9.0 failed with exit code 1.

* Try:
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to generate a Build Scan (Powered by Develocity).
> Get more help at https://help.gradle.org.

Deprecated Gradle features were used in this build, making it incompatible with Gradle 10.

You can use '--warning-mode all' to show the individual deprecation warnings and determine if they come from your own scripts or plugins.

For more on this, please refer to https://docs.gradle.org/9.1.0/userguide/command_line_interface.html#sec:command_line_warnings in the Gradle documentation.

BUILD FAILED in 3s
11 actionable tasks: 3 executed, 8 up-to-date
Finished stage 'native-image run' for 4.11.0 at 2026-05-08T06:49:57Z with exit code 1 after 4s
❌ ERROR: 'native-image run' for 4.11.0 failed with exit code 1.
SOFT-FAIL[native-image run][4.11.0][GVM_TCK_NATIVE_IMAGE_MODE="current-defaults" GVM_TCK_LV="4.11.0" ./gradlew clean test -Pcoordinates="org.seleniumhq.selenium:selenium-manager:4.9.0"]
========================================================================================
 org.seleniumhq.selenium:selenium-manager:4.9.0:4.11.0 stage javac compile
========================================================================================
Starting stage 'javac compile' for 4.11.0 at 2026-05-08T06:49:57Z
Command: GVM_TCK_NATIVE_IMAGE_MODE="current-defaults" GVM_TCK_LV="4.11.0" ./gradlew clean compileTestJava -Pcoordinates="org.seleniumhq.selenium:selenium-manager:4.9.0"
> Task :tck-build-logic:extractPluginRequests UP-TO-DATE
> Task :tck-build-logic:generatePluginAdapters UP-TO-DATE
> Task :tck-build-logic:compileJava UP-TO-DATE
> Task :tck-build-logic:compileGroovy UP-TO-DATE
> Task :tck-build-logic:compileGroovyPlugins UP-TO-DATE
> Task :tck-build-logic:pluginDescriptors UP-TO-DATE
> Task :tck-build-logic:processResources UP-TO-DATE
> Task :tck-build-logic:classes UP-TO-DATE
> Task :tck-build-logic:jar UP-TO-DATE

> Configure project :
GraalVM Reachability Metadata TCK
---------------------------------

> Task :tckClean
> Task :tck-build-logic:extractPluginRequests UP-TO-DATE
> Task :tck-build-logic:generatePluginAdapters UP-TO-DATE
> Task :tck-build-logic:compileJava UP-TO-DATE
> Task :tck-build-logic:compileGroovy UP-TO-DATE
> Task :tck-build-logic:compileGroovyPlugins UP-TO-DATE
> Task :tck-build-logic:pluginDescriptors UP-TO-DATE
> Task :tck-build-logic:processResources UP-TO-DATE
> Task :tck-build-logic:classes UP-TO-DATE
> Task :tck-build-logic:jar UP-TO-DATE
> Task :clean

[Incubating] Problems report is available at: file:///home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/tests/src/org.seleniumhq.selenium/selenium-manager/4.9.0/build/reports/problems/problems-report.html

Deprecated Gradle features were used in this build, making it incompatible with Gradle 10.

You can use '--warning-mode all' to show the individual deprecation warnings and determine if they come from your own scripts or plugins.

For more on this, please refer to https://docs.gradle.org/9.1.0/userguide/command_line_interface.html#sec:command_line_warnings in the Gradle documentation.

BUILD SUCCESSFUL in 824ms
9 actionable tasks: 1 executed, 8 up-to-date
Consider enabling configuration cache to speed up this build: https://docs.gradle.org/9.1.0/userguide/configuration_cache_enabling.html

> Task :clean

> Task :compileTestJava FAILED
> Task :tck-build-logic:extractPluginRequests UP-TO-DATE
> Task :tck-build-logic:generatePluginAdapters UP-TO-DATE
> Task :tck-build-logic:compileJava UP-TO-DATE
> Task :tck-build-logic:compileGroovy UP-TO-DATE
> Task :tck-build-logic:compileGroovyPlugins UP-TO-DATE
> Task :tck-build-logic:pluginDescriptors UP-TO-DATE
> Task :tck-build-logic:processResources UP-TO-DATE
> Task :tck-build-logic:classes UP-TO-DATE
> Task :tck-build-logic:jar UP-TO-DATE
> Task :tck-test-support:compileJava UP-TO-DATE
> Task :compileJava NO-SOURCE
> Task :processResources NO-SOURCE
> Task :classes UP-TO-DATE

> Task :compileTestJava FAILED
/home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/tests/src/org.seleniumhq.selenium/selenium-manager/4.9.0/src/test/java/org_seleniumhq_selenium/selenium_manager/SeleniumManagerTest.java:23: error: method getDriverPath in class SeleniumManager cannot be applied to given types;
[Incubating] Problems report is available at: file:///home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/tests/src/org.seleniumhq.selenium/selenium-manager/4.9.0/build/reports/problems/problems-report.html

Deprecated Gradle features were used in this build, making it incompatible with Gradle 10.

You can use '--warning-mode all' to show the individual deprecation warnings and determine if they come from your own scripts or plugins.

For more on this, please refer to https://docs.gradle.org/9.1.0/userguide/command_line_interface.html#sec:command_line_warnings in the Gradle documentation.
10 actionable tasks: 1 executed, 9 up-to-date
        assertThatThrownBy(() -> SeleniumManager.getInstance().getDriverPath(capabilities))
                                                              ^
  required: Capabilities,boolean
  found:    ImmutableCapabilities
  reason: actual and formal argument lists differ in length
1 error

FAILURE: Build failed with an exception.


* What went wrong:
Execution failed for task ':compileTestJava'.
> Compilation failed; see the compiler output below.
  /home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/tests/src/org.seleniumhq.selenium/selenium-manager/4.9.0/src/test/java/org_seleniumhq_selenium/selenium_manager/SeleniumManagerTest.java:23: error: method getDriverPath in class SeleniumManager cannot be applied to given types;
          assertThatThrownBy(() -> SeleniumManager.getInstance().getDriverPath(capabilities))
                                                                ^
    required: Capabilities,boolean
    found:    ImmutableCapabilities
    reason: actual and formal argument lists differ in length
  1 error

* Try:
> Check your code and dependencies to fix the compilation error(s)
> Run with --scan to generate a Build Scan (Powered by Develocity).

BUILD FAILED in 994ms

[Incubating] Problems report is available at: file:///home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/build/reports/problems/problems-report.html

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':compileTestJava'.
> Compilation failed

* Try:
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to generate a Build Scan (Powered by Develocity).
> Get more help at https://help.gradle.org.

Deprecated Gradle features were used in this build, making it incompatible with Gradle 10.

You can use '--warning-mode all' to show the individual deprecation warnings and determine if they come from your own scripts or plugins.

For more on this, please refer to https://docs.gradle.org/9.1.0/userguide/command_line_interface.html#sec:command_line_warnings in the Gradle documentation.

BUILD FAILED in 2s
11 actionable tasks: 3 executed, 8 up-to-date
Finished stage 'javac compile' for 4.11.0 at 2026-05-08T06:50:00Z with exit code 1 after 3s
❌ ERROR: 'javac compile' for 4.11.0 failed with exit code 1.
FAILED[javac compile][4.11.0][GVM_TCK_NATIVE_IMAGE_MODE="current-defaults" GVM_TCK_LV="4.11.0" ./gradlew clean compileTestJava -Pcoordinates="org.seleniumhq.selenium:selenium-manager:4.9.0"]
```

## javac compile on ubuntu-latest / JDK latest-ea / mode future-defaults-all

Failure kind: javac compile
OS: ubuntu-latest
JDK: latest-ea
Native-image mode: future-defaults-all
Reproducer: `GVM_TCK_NATIVE_IMAGE_MODE="future-defaults-all" GVM_TCK_LV="4.11.0" ./gradlew clean compileTestJava -Pcoordinates="org.seleniumhq.selenium:selenium-manager:4.9.0"`
Runner log: https://github.com/oracle/graalvm-reachability-metadata/actions/runs/25541257498/job/74967784555
Last 300 lines of the log:
```console
Test run finished after 7 ms
[         3 containers found      ]
[         0 containers skipped    ]
[         3 containers started    ]
[         0 containers aborted    ]
[         3 containers successful ]
[         0 containers failed     ]
[         1 tests found           ]
[         0 tests skipped         ]
[         1 tests started         ]
[         0 tests aborted         ]
[         1 tests successful      ]
[         0 tests failed          ]


[Incubating] Problems report is available at: file:///home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/tests/src/org.seleniumhq.selenium/selenium-manager/4.9.0/build/reports/problems/problems-report.html

Deprecated Gradle features were used in this build, making it incompatible with Gradle 10.

You can use '--warning-mode all' to show the individual deprecation warnings and determine if they come from your own scripts or plugins.

For more on this, please refer to https://docs.gradle.org/9.1.0/userguide/command_line_interface.html#sec:command_line_warnings in the Gradle documentation.

BUILD SUCCESSFUL in 1m 22s
17 actionable tasks: 7 executed, 10 up-to-date
Consider enabling configuration cache to speed up this build: https://docs.gradle.org/9.1.0/userguide/configuration_cache_enabling.html
-------
Test for org.seleniumhq.selenium:selenium-manager:4.9.0 passed.
====================

> Task :test

[Incubating] Problems report is available at: file:///home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/build/reports/problems/problems-report.html

Deprecated Gradle features were used in this build, making it incompatible with Gradle 10.

You can use '--warning-mode all' to show the individual deprecation warnings and determine if they come from your own scripts or plugins.

For more on this, please refer to https://docs.gradle.org/9.1.0/userguide/command_line_interface.html#sec:command_line_warnings in the Gradle documentation.

BUILD SUCCESSFUL in 1m 24s
11 actionable tasks: 3 executed, 8 up-to-date
Finished stage 'native-image run' for 4.10.0 at 2026-05-08T06:49:49Z with exit code 0 after 85s
PASSED:4.10.0
========================================================================================
 Testing org.seleniumhq.selenium:selenium-manager:4.9.0:4.11.0
========================================================================================
========================================================================================
 org.seleniumhq.selenium:selenium-manager:4.9.0:4.11.0 stage native-image run
========================================================================================
Starting stage 'native-image run' for 4.11.0 at 2026-05-08T06:49:49Z
Command: GVM_TCK_NATIVE_IMAGE_MODE="future-defaults-all" GVM_TCK_LV="4.11.0" ./gradlew clean test -Pcoordinates="org.seleniumhq.selenium:selenium-manager:4.9.0"
> Task :tck-build-logic:extractPluginRequests UP-TO-DATE
> Task :tck-build-logic:generatePluginAdapters UP-TO-DATE
> Task :tck-build-logic:compileJava UP-TO-DATE
> Task :tck-build-logic:compileGroovy UP-TO-DATE
> Task :tck-build-logic:compileGroovyPlugins UP-TO-DATE
> Task :tck-build-logic:pluginDescriptors UP-TO-DATE
> Task :tck-build-logic:processResources UP-TO-DATE
> Task :tck-build-logic:classes UP-TO-DATE
> Task :tck-build-logic:jar UP-TO-DATE

> Configure project :
GraalVM Reachability Metadata TCK
---------------------------------

> Task :tckClean
> Task :tck-build-logic:extractPluginRequests UP-TO-DATE
> Task :tck-build-logic:generatePluginAdapters UP-TO-DATE
> Task :tck-build-logic:compileJava UP-TO-DATE
> Task :tck-build-logic:compileGroovy UP-TO-DATE
> Task :tck-build-logic:compileGroovyPlugins UP-TO-DATE
> Task :tck-build-logic:pluginDescriptors UP-TO-DATE
> Task :tck-build-logic:processResources UP-TO-DATE
> Task :tck-build-logic:classes UP-TO-DATE
> Task :tck-build-logic:jar UP-TO-DATE
> Task :clean

[Incubating] Problems report is available at: file:///home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/tests/src/org.seleniumhq.selenium/selenium-manager/4.9.0/build/reports/problems/problems-report.html

Deprecated Gradle features were used in this build, making it incompatible with Gradle 10.

You can use '--warning-mode all' to show the individual deprecation warnings and determine if they come from your own scripts or plugins.

For more on this, please refer to https://docs.gradle.org/9.1.0/userguide/command_line_interface.html#sec:command_line_warnings in the Gradle documentation.

BUILD SUCCESSFUL in 868ms
9 actionable tasks: 1 executed, 8 up-to-date
Consider enabling configuration cache to speed up this build: https://docs.gradle.org/9.1.0/userguide/configuration_cache_enabling.html

> Task :clean

> Task :tckTest FAILED
====================
Testing library: org.seleniumhq.selenium:selenium-manager:4.9.0
Command: `/home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/gradlew nativeTest`
Executing test...
-------
> Task :tck-build-logic:extractPluginRequests UP-TO-DATE
> Task :tck-build-logic:generatePluginAdapters UP-TO-DATE
> Task :tck-build-logic:compileJava UP-TO-DATE
> Task :tck-build-logic:compileGroovy UP-TO-DATE
> Task :tck-build-logic:compileGroovyPlugins UP-TO-DATE
> Task :tck-build-logic:pluginDescriptors UP-TO-DATE
> Task :tck-build-logic:processResources UP-TO-DATE
> Task :tck-build-logic:classes UP-TO-DATE
> Task :tck-build-logic:jar UP-TO-DATE
> Task :compileJava NO-SOURCE
> Task :tck-test-support:compileJava UP-TO-DATE
> Task :processResources NO-SOURCE
> Task :tck-test-support:processResources NO-SOURCE
> Task :classes UP-TO-DATE
> Task :tck-test-support:classes UP-TO-DATE
> Task :tck-test-support:jar UP-TO-DATE

> Task :compileTestJava FAILED
/home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/tests/src/org.seleniumhq.selenium/selenium-manager/4.9.0/src/test/java/org_seleniumhq_selenium/selenium_manager/SeleniumManagerTest.java:23: error: method getDriverPath in class SeleniumManager cannot be applied to given types;
        assertThatThrownBy(() -> SeleniumManager.getInstance().getDriverPath(capabilities))
                                                              ^
  required: Capabilities,boolean
  found:    ImmutableCapabilities
  reason: actual and formal argument lists differ in length
1 error

FAILURE: Build failed with an exception.

* What went wrong:
[Incubating] Problems report is available at: file:///home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/tests/src/org.seleniumhq.selenium/selenium-manager/4.9.0/build/reports/problems/problems-report.html

Execution failed for task ':compileTestJava'.
> Compilation failed; see the compiler output below.
  /home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/tests/src/org.seleniumhq.selenium/selenium-manager/4.9.0/src/test/java/org_seleniumhq_selenium/selenium_manager/SeleniumManagerTest.java:23: error: method getDriverPath in class SeleniumManager cannot be applied to given types;
          assertThatThrownBy(() -> SeleniumManager.getInstance().getDriverPath(capabilities))
                                                                ^
    required: Capabilities,boolean
    found:    ImmutableCapabilities
    reason: actual and formal argument lists differ in length
  1 error

* Try:
> Check your code and dependencies to fix the compilation error(s)
> Run with --scan to generate a Build Scan (Powered by Develocity).

Deprecated Gradle features were used in this build, making it incompatible with Gradle 10.

You can use '--warning-mode all' to show the individual deprecation warnings and determine if they come from your own scripts or plugins.

For more on this, please refer to https://docs.gradle.org/9.1.0/userguide/command_line_interface.html#sec:command_line_warnings in the Gradle documentation.

BUILD FAILED in 1s
11 actionable tasks: 1 executed, 10 up-to-date

[Incubating] Problems report is available at: file:///home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/build/reports/problems/problems-report.html

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':tckTest'.
> Test for org.seleniumhq.selenium:selenium-manager:4.9.0 failed with exit code 1.

* Try:
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to generate a Build Scan (Powered by Develocity).
> Get more help at https://help.gradle.org.

Deprecated Gradle features were used in this build, making it incompatible with Gradle 10.

You can use '--warning-mode all' to show the individual deprecation warnings and determine if they come from your own scripts or plugins.

For more on this, please refer to https://docs.gradle.org/9.1.0/userguide/command_line_interface.html#sec:command_line_warnings in the Gradle documentation.

BUILD FAILED in 3s
11 actionable tasks: 3 executed, 8 up-to-date
Finished stage 'native-image run' for 4.11.0 at 2026-05-08T06:49:52Z with exit code 1 after 3s
❌ ERROR: 'native-image run' for 4.11.0 failed with exit code 1.
SOFT-FAIL[native-image run][4.11.0][GVM_TCK_NATIVE_IMAGE_MODE="future-defaults-all" GVM_TCK_LV="4.11.0" ./gradlew clean test -Pcoordinates="org.seleniumhq.selenium:selenium-manager:4.9.0"]
========================================================================================
 org.seleniumhq.selenium:selenium-manager:4.9.0:4.11.0 stage javac compile
========================================================================================
Starting stage 'javac compile' for 4.11.0 at 2026-05-08T06:49:52Z
Command: GVM_TCK_NATIVE_IMAGE_MODE="future-defaults-all" GVM_TCK_LV="4.11.0" ./gradlew clean compileTestJava -Pcoordinates="org.seleniumhq.selenium:selenium-manager:4.9.0"
> Task :tck-build-logic:extractPluginRequests UP-TO-DATE
> Task :tck-build-logic:generatePluginAdapters UP-TO-DATE
> Task :tck-build-logic:compileJava UP-TO-DATE
> Task :tck-build-logic:compileGroovy UP-TO-DATE
> Task :tck-build-logic:compileGroovyPlugins UP-TO-DATE
> Task :tck-build-logic:pluginDescriptors UP-TO-DATE
> Task :tck-build-logic:processResources UP-TO-DATE
> Task :tck-build-logic:classes UP-TO-DATE
> Task :tck-build-logic:jar UP-TO-DATE

> Configure project :
GraalVM Reachability Metadata TCK
---------------------------------

> Task :tckClean
> Task :tck-build-logic:extractPluginRequests UP-TO-DATE
> Task :tck-build-logic:generatePluginAdapters UP-TO-DATE
> Task :tck-build-logic:compileJava UP-TO-DATE
> Task :tck-build-logic:compileGroovy UP-TO-DATE
> Task :tck-build-logic:compileGroovyPlugins UP-TO-DATE
> Task :tck-build-logic:pluginDescriptors UP-TO-DATE
> Task :tck-build-logic:processResources UP-TO-DATE
> Task :tck-build-logic:classes UP-TO-DATE
> Task :tck-build-logic:jar UP-TO-DATE
> Task :clean

[Incubating] Problems report is available at: file:///home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/tests/src/org.seleniumhq.selenium/selenium-manager/4.9.0/build/reports/problems/problems-report.html

Deprecated Gradle features were used in this build, making it incompatible with Gradle 10.

You can use '--warning-mode all' to show the individual deprecation warnings and determine if they come from your own scripts or plugins.

For more on this, please refer to https://docs.gradle.org/9.1.0/userguide/command_line_interface.html#sec:command_line_warnings in the Gradle documentation.

BUILD SUCCESSFUL in 812ms
9 actionable tasks: 1 executed, 8 up-to-date
Consider enabling configuration cache to speed up this build: https://docs.gradle.org/9.1.0/userguide/configuration_cache_enabling.html

> Task :clean

> Task :compileTestJava FAILED
> Task :tck-build-logic:extractPluginRequests UP-TO-DATE
> Task :tck-build-logic:generatePluginAdapters UP-TO-DATE
> Task :tck-build-logic:compileJava UP-TO-DATE
> Task :tck-build-logic:compileGroovy UP-TO-DATE
> Task :tck-build-logic:compileGroovyPlugins UP-TO-DATE
> Task :tck-build-logic:pluginDescriptors UP-TO-DATE
> Task :tck-build-logic:processResources UP-TO-DATE
> Task :tck-build-logic:classes UP-TO-DATE
> Task :tck-build-logic:jar UP-TO-DATE
> Task :tck-test-support:compileJava UP-TO-DATE
> Task :compileJava NO-SOURCE
> Task :processResources NO-SOURCE
> Task :classes UP-TO-DATE

> Task :compileTestJava FAILED
/home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/tests/src/org.seleniumhq.selenium/selenium-manager/4.9.0/src/test/java/org_seleniumhq_selenium/selenium_manager/SeleniumManagerTest.java:23: error: method getDriverPath in class SeleniumManager cannot be applied to given types;
        assertThatThrownBy(() -> SeleniumManager.getInstance().getDriverPath(capabilities))
                                                              ^
  required: Capabilities,boolean
  found:    ImmutableCapabilities
  reason: actual and formal argument lists differ in length
1 error

[Incubating] Problems report is available at: file:///home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/tests/src/org.seleniumhq.selenium/selenium-manager/4.9.0/build/reports/problems/problems-report.html

Deprecated Gradle features were used in this build, making it incompatible with Gradle 10.

You can use '--warning-mode all' to show the individual deprecation warnings and determine if they come from your own scripts or plugins.

For more on this, please refer to https://docs.gradle.org/9.1.0/userguide/command_line_interface.html#sec:command_line_warnings in the Gradle documentation.
10 actionable tasks: 1 executed, 9 up-to-date

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':compileTestJava'.
> Compilation failed; see the compiler output below.
  /home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/tests/src/org.seleniumhq.selenium/selenium-manager/4.9.0/src/test/java/org_seleniumhq_selenium/selenium_manager/SeleniumManagerTest.java:23: error: method getDriverPath in class SeleniumManager cannot be applied to given types;
          assertThatThrownBy(() -> SeleniumManager.getInstance().getDriverPath(capabilities))
                                                                ^
    required: Capabilities,boolean
    found:    ImmutableCapabilities
    reason: actual and formal argument lists differ in length
  1 error

* Try:
> Check your code and dependencies to fix the compilation error(s)
> Run with --scan to generate a Build Scan (Powered by Develocity).

BUILD FAILED in 901ms

[Incubating] Problems report is available at: file:///home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/build/reports/problems/problems-report.html

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':compileTestJava'.
> Compilation failed

* Try:
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to generate a Build Scan (Powered by Develocity).
> Get more help at https://help.gradle.org.

Deprecated Gradle features were used in this build, making it incompatible with Gradle 10.

You can use '--warning-mode all' to show the individual deprecation warnings and determine if they come from your own scripts or plugins.

For more on this, please refer to https://docs.gradle.org/9.1.0/userguide/command_line_interface.html#sec:command_line_warnings in the Gradle documentation.

BUILD FAILED in 2s
11 actionable tasks: 3 executed, 8 up-to-date
Finished stage 'javac compile' for 4.11.0 at 2026-05-08T06:49:55Z with exit code 1 after 3s
❌ ERROR: 'javac compile' for 4.11.0 failed with exit code 1.
FAILED[javac compile][4.11.0][GVM_TCK_NATIVE_IMAGE_MODE="future-defaults-all" GVM_TCK_LV="4.11.0" ./gradlew clean compileTestJava -Pcoordinates="org.seleniumhq.selenium:selenium-manager:4.9.0"]
```

--- a/metadata/org.seleniumhq.selenium/selenium-manager/4.11.0/reachability-metadata.json
+++ b/metadata/org.seleniumhq.selenium/selenium-manager/4.11.0/reachability-metadata.json
@@ -1,0 +1,131 @@
+{
+  "reflection" : [
+    {
+      "condition" : {
+        "typeReached" : "org.openqa.selenium.manager.SeleniumManager"
+      },
+      "type" : "org.openqa.selenium.manager.SeleniumManagerOutput",
+      "methods" : [
+        {
+          "name" : "<init>",
+          "parameterTypes" : [ ]
+        },
+        {
+          "name" : "setLogs",
+          "parameterTypes" : [
+            "java.util.List"
+          ]
+        },
+        {
+          "name" : "setResult",
+          "parameterTypes" : [
+            "org.openqa.selenium.manager.SeleniumManagerOutput$Result"
+          ]
+        }
+      ]
+    },
+    {
+      "condition" : {
+        "typeReached" : "org.openqa.selenium.manager.SeleniumManager"
+      },
+      "type" : "org.openqa.selenium.manager.SeleniumManagerOutput$Log",
+      "methods" : [
+        {
+          "name" : "<init>",
+          "parameterTypes" : [ ]
+        },
+        {
+          "name" : "setLevel",
+          "parameterTypes" : [
+            "java.lang.String"
+          ]
+        },
+        {
+          "name" : "setTimestamp",
+          "parameterTypes" : [
+            "long"
+          ]
+        },
+        {
+          "name" : "setMessage",
+          "parameterTypes" : [
+            "java.lang.String"
+          ]
+        }
+      ]
+    },
+    {
+      "condition" : {
+        "typeReached" : "org.openqa.selenium.manager.SeleniumManager"
+      },
+      "type" : "org.openqa.selenium.manager.SeleniumManagerOutput$Result",
+      "methods" : [
+        {
+          "name" : "<init>",
+          "parameterTypes" : [
+            "java.lang.String"
+          ]
+        },
+        {
+          "name" : "<init>",
+          "parameterTypes" : [
+            "int",
+            "java.lang.String",
+            "java.lang.String",
+            "java.lang.String"
+          ]
+        },
+        {
+          "name" : "setCode",
+          "parameterTypes" : [
+            "int"
+          ]
+        },
+        {
+          "name" : "setMessage",
+          "parameterTypes" : [
+            "java.lang.String"
+          ]
+        },
+        {
+          "name" : "setDriverPath",
+          "parameterTypes" : [
+            "java.lang.String"
+          ]
+        },
+        {
+          "name" : "setBrowserPath",
+          "parameterTypes" : [
+            "java.lang.String"
+          ]
+        },
+        {
+          "name" : "fromJson",
+          "parameterTypes" : [
+            "org.openqa.selenium.json.JsonInput"
+          ]
+        }
+      ]
+    }
+  ],
+  "resources" : [
+    {
+      "condition" : {
+        "typeReached" : "org.openqa.selenium.manager.SeleniumManager"
+      },
+      "glob" : "org/openqa/selenium/manager/linux/selenium-manager"
+    },
+    {
+      "condition" : {
+        "typeReached" : "org.openqa.selenium.manager.SeleniumManager"
+      },
+      "glob" : "org/openqa/selenium/manager/macos/selenium-manager"
+    },
+    {
+      "condition" : {
+        "typeReached" : "org.openqa.selenium.manager.SeleniumManager"
+      },
+      "glob" : "org/openqa/selenium/manager/windows/selenium-manager.exe"
+    }
+  ]
+}

--- a/metadata/org.seleniumhq.selenium/selenium-manager/index.json
+++ b/metadata/org.seleniumhq.selenium/selenium-manager/index.json
@@ -13,5 +13,14 @@
     "allowed-packages" : [
       "org.openqa.selenium.manager"
     ]
+  },
+  {
+    "metadata-version" : "4.11.0",
+    "tested-versions" : [
+      "4.11.0"
+    ],
+    "allowed-packages" : [
+      "org.openqa.selenium.manager"
+    ]
   }
 ]

--- a/metadata/org.seleniumhq.selenium/selenium-manager/index.json
+++ b/metadata/org.seleniumhq.selenium/selenium-manager/index.json
@@ -16,6 +16,11 @@
   },
   {
     "metadata-version" : "4.11.0",
+    "source-code-url" : "https://repo.maven.apache.org/maven2/org/seleniumhq/selenium/selenium-manager/$version$/selenium-manager-$version$-sources.jar",
+    "repository-url" : "https://github.com/SeleniumHQ/selenium/",
+    "test-code-url" : "https://github.com/SeleniumHQ/selenium/tree/selenium-$version$/rust/tests",
+    "documentation-url" : "https://repo.maven.apache.org/maven2/org/seleniumhq/selenium/selenium-manager/$version$/selenium-manager-$version$-javadoc.jar",
+    "description" : "Selenium Manager provides the Java binding wrapper and bundled platform executables for Selenium's browser and driver management tool. It locates or downloads the browser driver required for Selenium WebDriver sessions and exposes the resolved driver path to the Java Selenium stack.",
     "tested-versions" : [
       "4.11.0"
     ],

--- a/stats/org.seleniumhq.selenium/selenium-manager/4.11.0/execution-metrics.json
+++ b/stats/org.seleniumhq.selenium/selenium-manager/4.11.0/execution-metrics.json
@@ -1,0 +1,99 @@
+{
+  "fix_javac_fail:2026-05-08": {
+    "timestamp": "2026-05-08T09:01:23.263619Z",
+    "library": "org.seleniumhq.selenium:selenium-manager:4.11.0",
+    "starting_commit": "56662e3342385fb135a4b4b7923a7bde4266877c",
+    "ending_commit": "10d85d5a58d49288697c34539c9b6c434653a792",
+    "previous_library": "org.seleniumhq.selenium:selenium-manager:4.9.0",
+    "strategy_name": "javac_iterative_with_coverage_sources_pi_gpt-5.5",
+    "agent": "pi",
+    "model": "oca/gpt-5.5",
+    "status": "success",
+    "stats": {
+      "version": "4.11.0",
+      "dynamicAccess": {
+        "breakdown": {
+          "resources": {
+            "coverageRatio": 1.0,
+            "coveredCalls": 1,
+            "totalCalls": 1
+          }
+        },
+        "coverageRatio": 1.0,
+        "coveredCalls": 1,
+        "totalCalls": 1
+      },
+      "libraryCoverage": {
+        "instruction": {
+          "covered": 452,
+          "missed": 259,
+          "ratio": 0.635724,
+          "total": 711
+        },
+        "line": {
+          "covered": 116,
+          "missed": 70,
+          "ratio": 0.623656,
+          "total": 186
+        },
+        "method": {
+          "covered": 19,
+          "missed": 14,
+          "ratio": 0.575758,
+          "total": 33
+        }
+      }
+    },
+    "previous_library_stats": {
+      "version": "4.9.0",
+      "dynamicAccess": {
+        "breakdown": {
+          "resources": {
+            "coverageRatio": 1.0,
+            "coveredCalls": 1,
+            "totalCalls": 1
+          }
+        },
+        "coverageRatio": 1.0,
+        "coveredCalls": 1,
+        "totalCalls": 1
+      },
+      "libraryCoverage": {
+        "instruction": {
+          "covered": 298,
+          "missed": 184,
+          "ratio": 0.618257,
+          "total": 482
+        },
+        "line": {
+          "covered": 71,
+          "missed": 36,
+          "ratio": 0.663551,
+          "total": 107
+        },
+        "method": {
+          "covered": 18,
+          "missed": 9,
+          "ratio": 0.666667,
+          "total": 27
+        }
+      }
+    },
+    "metrics": {
+      "input_tokens_used": 18862,
+      "cached_input_tokens_used": 41984,
+      "output_tokens_used": 722,
+      "iterations": 1,
+      "cost_usd": 0.0427,
+      "tested_library_loc": 116,
+      "metadata_entries": 20,
+      "previous_library_metadata_entries": 16,
+      "code_coverage_percent": 62.37,
+      "previous_library_coverage_percent": 66.36
+    },
+    "artifacts": {
+      "test_file": "tests/src/org.seleniumhq.selenium/selenium-manager/4.11.0/src/test/java/org_seleniumhq_selenium/selenium_manager/SeleniumManagerTest.java",
+      "metadata_file": "metadata/org.seleniumhq.selenium/selenium-manager/4.11.0/reachability-metadata.json"
+    }
+  }
+}

--- a/stats/org.seleniumhq.selenium/selenium-manager/4.11.0/stats.json
+++ b/stats/org.seleniumhq.selenium/selenium-manager/4.11.0/stats.json
@@ -1,0 +1,37 @@
+{
+  "versions" : [ {
+    "version" : "4.11.0",
+    "dynamicAccess" : {
+      "breakdown" : {
+        "resources" : {
+          "coverageRatio" : 1.0,
+          "coveredCalls" : 1,
+          "totalCalls" : 1
+        }
+      },
+      "coverageRatio" : 1.0,
+      "coveredCalls" : 1,
+      "totalCalls" : 1
+    },
+    "libraryCoverage" : {
+      "instruction" : {
+        "covered" : 452,
+        "missed" : 259,
+        "ratio" : 0.635724,
+        "total" : 711
+      },
+      "line" : {
+        "covered" : 116,
+        "missed" : 70,
+        "ratio" : 0.623656,
+        "total" : 186
+      },
+      "method" : {
+        "covered" : 19,
+        "missed" : 14,
+        "ratio" : 0.575758,
+        "total" : 33
+      }
+    }
+  } ]
+}

--- a/tests/src/org.seleniumhq.selenium/selenium-manager/4.11.0/.gitignore
+++ b/tests/src/org.seleniumhq.selenium/selenium-manager/4.11.0/.gitignore
@@ -1,0 +1,4 @@
+gradlew.bat
+gradlew
+gradle/
+build/

--- a/tests/src/org.seleniumhq.selenium/selenium-manager/4.11.0/build.gradle
+++ b/tests/src/org.seleniumhq.selenium/selenium-manager/4.11.0/build.gradle
@@ -1,0 +1,27 @@
+/*
+ * Copyright and related rights waived via CC0
+ *
+ * You should have received a copy of the CC0 legalcode along with this
+ * work. If not, see <http://creativecommons.org/publicdomain/zero/1.0/>.
+ */
+plugins {
+    id "org.graalvm.internal.tck"
+}
+
+String libraryVersion = tck.testedLibraryVersion.get()
+
+dependencies {
+    testImplementation "org.seleniumhq.selenium:selenium-manager:$libraryVersion"
+    testImplementation 'org.assertj:assertj-core:3.22.0'
+}
+
+graalvmNative {
+    agent {
+        defaultMode = "conditional"
+        modes {
+            conditional {
+                userCodeFilterPath = "user-code-filter.json"
+            }
+        }
+    }
+}

--- a/tests/src/org.seleniumhq.selenium/selenium-manager/4.11.0/gradle.properties
+++ b/tests/src/org.seleniumhq.selenium/selenium-manager/4.11.0/gradle.properties
@@ -1,0 +1,6 @@
+# Optional explicit tested library coordinates (format: group:artifact:version).
+# If omitted, fallback resolution in org.graalvm.internal.tck.gradle still uses
+# environment variable GVM_TCK_LC and then this property.
+library.coordinates = org.seleniumhq.selenium:selenium-manager:4.11.0
+library.version = 4.11.0
+metadata.dir = org.seleniumhq.selenium/selenium-manager/4.11.0/

--- a/tests/src/org.seleniumhq.selenium/selenium-manager/4.11.0/settings.gradle
+++ b/tests/src/org.seleniumhq.selenium/selenium-manager/4.11.0/settings.gradle
@@ -1,0 +1,20 @@
+/*
+ * Copyright and related rights waived via CC0
+ *
+ * You should have received a copy of the CC0 legalcode along with this
+ * work. If not, see <http://creativecommons.org/publicdomain/zero/1.0/>.
+ */
+pluginManagement {
+    def tckPath = Objects.requireNonNullElse(
+            System.getenv("GVM_TCK_TCKDIR"),
+            "../../../../tck-build-logic"
+    )
+    includeBuild(tckPath)
+    repositories { mavenLocal(); gradlePluginPortal() }
+}
+
+plugins {
+    id "org.graalvm.internal.tck-settings" version "1.0.0-SNAPSHOT"
+}
+
+rootProject.name = 'org.seleniumhq.selenium.selenium-manager_tests'

--- a/tests/src/org.seleniumhq.selenium/selenium-manager/4.11.0/src/test/java/org_seleniumhq_selenium/selenium_manager/SeleniumManagerTest.java
+++ b/tests/src/org.seleniumhq.selenium/selenium-manager/4.11.0/src/test/java/org_seleniumhq_selenium/selenium_manager/SeleniumManagerTest.java
@@ -1,0 +1,27 @@
+/*
+ * Copyright and related rights waived via CC0
+ *
+ * You should have received a copy of the CC0 legalcode along with this
+ * work. If not, see <http://creativecommons.org/publicdomain/zero/1.0/>.
+ */
+package org_seleniumhq_selenium.selenium_manager;
+
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.Timeout;
+import org.openqa.selenium.ImmutableCapabilities;
+import org.openqa.selenium.WebDriverException;
+import org.openqa.selenium.manager.SeleniumManager;
+
+public class SeleniumManagerTest {
+    @Test
+    @Timeout(30)
+    void extractsBundledManagerBinaryBeforeRunningBrowserResolution() {
+        ImmutableCapabilities capabilities = new ImmutableCapabilities("browserName", "unknown-browser-for-test");
+
+        assertThatThrownBy(() -> SeleniumManager.getInstance().getDriverPath(capabilities))
+                .isInstanceOf(WebDriverException.class)
+                .hasMessageContaining("Invalid browser name: unknown-browser-for-test");
+    }
+}

--- a/tests/src/org.seleniumhq.selenium/selenium-manager/4.11.0/src/test/java/org_seleniumhq_selenium/selenium_manager/SeleniumManagerTest.java
+++ b/tests/src/org.seleniumhq.selenium/selenium-manager/4.11.0/src/test/java/org_seleniumhq_selenium/selenium_manager/SeleniumManagerTest.java
@@ -20,7 +20,7 @@ public class SeleniumManagerTest {
     void extractsBundledManagerBinaryBeforeRunningBrowserResolution() {
         ImmutableCapabilities capabilities = new ImmutableCapabilities("browserName", "unknown-browser-for-test");
 
-        assertThatThrownBy(() -> SeleniumManager.getInstance().getDriverPath(capabilities))
+        assertThatThrownBy(() -> SeleniumManager.getInstance().getDriverPath(capabilities, false))
                 .isInstanceOf(WebDriverException.class)
                 .hasMessageContaining("Invalid browser name: unknown-browser-for-test");
     }

--- a/tests/src/org.seleniumhq.selenium/selenium-manager/4.11.0/user-code-filter.json
+++ b/tests/src/org.seleniumhq.selenium/selenium-manager/4.11.0/user-code-filter.json
@@ -1,0 +1,13 @@
+{
+  "rules" : [
+    {
+      "excludeClasses" : "**"
+    },
+    {
+      "includeClasses" : "org.openqa.selenium.manager.**"
+    },
+    {
+      "includeClasses" : "org_seleniumhq_selenium.selenium_manager.**"
+    }
+  ]
+}

